cypress/test/navigation/MobileNavMenu.cy.js

```diff
@@ -0,0 +1,7 @@
+import { mount } from 'cypress/react'
+import MobileNavMenu from '../../../components/navigation/MobileNavMenu';
+describe('StickyNavbar', () => {
+    it('renders the navbar with children and custom class', () => {
+        mount(<MobileNavMenu ></MobileNavMenu>);
+    });
```

Comment:
You have to check each Link and it's respective logo beside it, using `data-testid`.

Reply:
done